docs/documentation/zh/declaration-files/By Example.md

```diff
@@ -1,24 +1,21 @@
 ---
-title: Declaration Reference
+title: 声明文件参考
 layout: docs
 permalink: /zh/docs/handbook/declaration-files/by-example.html
-oneline: "How to create a d.ts file for a module"
+oneline: "如何为一个模块创建一个 d.ts 文件"
```

Comment:
```suggestion
oneline: "如何为模块创建 d.ts 文件"
```

Reply:
Sometimes, it may be better not to translate the word "a" so that the translation could be more readable.

Reply:
from my perspective it's a personal preference, but we can make it a conventional rule, what do you think @awxiaoxian2020

Reply:
I agree with @hanyujie2002. It depends on context. I don't think we can have a rule to describe how we deal with "article". In Chinese, we only have "measure word" (The "article" will be translated to it). But the "article" serves grammar more instead of "single". We should translate "article" when it is addressing "single".